### DISCUSSION TITLE
Welcome to meshpy Discussions!

### DISCUSSION THREAD

**Discussion by inducer** [inducer/meshpy#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AminMhmd** [comment]
hello! one quick question. Is the mesh.elements that give the node numbers of each element used counterclockwise?

**Reply by inducer** [reply]
Please start a new thread in Q&A.
